<launch>

    <arg name="conf_file" default="$(find collision_free_vel)/conf/config.yaml"/>
   
    <!-- Launch collision avoidance algorithm -->
    
    <node pkg="collision_free_vel" type="central_avoidance_node.py" name="central_controller" output="screen">
        <rosparam command="load" file="$(arg conf_file)" />
        <param name="uav_radius" type="double" value="3.0" />
        <param name="n_directions" type="int" value="10" />
        <param name="time_horizon" type="double" value="10" />
        <param name="coll_detection_time" type="double" value="5" />
        <param name="max_deviation" type="double" value="1.0" />
        <param name="rate" type="double" value="10" />
        <param name="nominal_speed" type="double" value="1.0" />
    </node>
    
</launch>
